<launch>
  <node pkg="fixed_point_cloud_mapping" type="pcd_to_msg_node" name="pcd_to_msg_node" output="screen" respawn="true"
	args="$(optenv pcd /home/pangerci/work/iros_cad/hokuyo/pfh_denoise/teapot_handle_side-denoise.pcd) $(optenv period 5)" />

  <node pkg="cloud_algos" type="statistical_noise_removal_node" name="statistical_noise_removal_node" output="screen" respawn="true">
    <remap from="~cloud_pcd" to="cloud_pcd" />
    <rosparam>{alpha: 1, neighborhood_size: 30}</rosparam>
  </node>
</launch>
